dev/io.openliberty.microprofile.openapi.4.0.internal/bnd.bnd

```diff
@@ -0,0 +1,68 @@
+#*******************************************************************************
+# Copyright (c) 2024, 2025 IBM Corporation and others.
```

Comment:
Copied from `io.openliberty.microprofile.openapi.4.0.internal.services/bnd.bnd` since most of the code from there has been moved to this new project.